packages/crayons-core/src/components/icon/library.system.ts

```diff
@@ -0,0 +1,91 @@
+  'success':
+    "<svg viewBox='0 0 16 16'><path d='M8 0a8 8 0 1 0 5.657 2.343A8 8 0 0 0 8 0zm0 14.6A6.6 6.6 0 1 1 14.6 8 6.6 6.6 0 0 1 8 14.6zm3.79-9.27a.69.69 0 0 1 0 1l-4.34 4.34a.71.71 0 0 1-1 0L4.21 8.41a.707.707 0 0 1 1-1L7 9.18l3.8-3.85a.71.71 0 0 1 .99 0z' fill='#00a886'/></svg>",
+  'ticket-primary':
+    "<svg viewBox='0 0 16 16'><path d='M16 8a.72.72 0 0 1-.38.56l-1.5 1.11.74 1.7a.7.7 0 0 1 0 .63.73.73 0 0 1-.53.35l-1.85.2-.2 1.85a.73.73 0 0 1-.35.53.7.7 0 0 1-.63 0l-1.7-.74-1.11 1.49a.69.69 0 0 1-1.12 0l-1.11-1.49-1.7.74a.7.7 0 0 1-.63 0 .73.73 0 0 1-.35-.53l-.2-1.85-1.85-.2A.73.73 0 0 1 1 12a.7.7 0 0 1 0-.63l.78-1.7-1.5-1.11a.7.7 0 0 1 0-1.12l1.53-1.11-.74-1.7a.7.7 0 0 1 0-.63.73.73 0 0 1 .53-.42l1.85-.2.2-1.85A.73.73 0 0 1 4 1a.7.7 0 0 1 .63 0l1.7.78L7.44.29a.72.72 0 0 1 1.12 0l1.11 1.52 1.7-.74a.7.7 0 0 1 .63 0 .73.73 0 0 1 .42.53l.2 1.85 1.85.2A.73.73 0 0 1 15 4a.7.7 0 0 1 0 .63l-.78 1.7 1.5 1.11A.72.72 0 0 1 16 8zm-3.29 1.72a.7.7 0 0 1 .23-.85L14.12 8l-1.18-.87a.7.7 0 0 1-.23-.85l.59-1.34-1.46-.16a.72.72 0 0 1-.62-.62l-.16-1.46-1.34.59a.72.72 0 0 1-.85-.23L8 1.88l-.87 1.18a.71.71 0 0 1-.85.23L4.94 2.7l-.16 1.46a.72.72 0 0 1-.62.62l-1.46.16.59 1.34a.7.7 0 0 1-.23.85L1.88 8l1.18.87a.7.7 0 0 1 .23.85l-.59 1.34 1.46.16a.72.72 0 0 1 .62.62l.16 1.46 1.34-.59a.7.7 0 0 1 .85.23L8 14.12l.87-1.18a.72.72 0 0 1 .85-.23l1.34.59.16-1.46a.72.72 0 0 1 .62-.62l1.46-.16-.59-1.34zM9.8 6a.71.71 0 0 1 .97 0 .69.69 0 0 1 0 1l-3 3a.71.71 0 0 1-.99 0L5.2 8.44a.69.69 0 0 1 0-1 .71.71 0 0 1 1 0l1.07 1.08L9.8 6z' fill='currentColor' fill-rule='evenodd'/></svg>",
+  'verified':
+    "<svg viewBox='0 0 17 17'><path d='M13.244 13.052H2.877a.8.8 0 0 1-.777-.629L.068 3.054A.801.801 0 0 1 1.3 2.22l3.22 2.22L7.41.342a.81.81 0 0 1 1.301 0L11.6 4.438l3.197-2.219a.801.801 0 0 1 1.233.835l-2.01 9.37a.8.8 0 0 1-.776.628zm-9.75-1.613h9.134l1.415-6.749-2.192 1.522a.787.787 0 0 1-1.142-.206L8.06 2.196l-2.683 3.81a.788.788 0 0 1-1.142.206L2.077 4.69l1.416 6.75zm9.784 4.576H2.842a.8.8 0 0 1 0-1.601h10.436a.8.8 0 0 1 0 1.601z' fill='currentColor' fill-rule='evenodd'/></svg>",
+  'vertical-align-bottom':
+    "<svg viewBox='0 0 16 16'><path d='M15.3 14.5H14V8.8a.7.7 0 0 0-.7-.7H9.7a.7.7 0 0 0-.7.7v5.7H7V.8a.7.7 0 0 0-.7-.7H2.7a.7.7 0 0 0-.7.7v13.7H.7a.7.7 0 0 0 0 1.4h14.6a.7.7 0 0 0 0-1.4zm-11.9 0v-13h2.2v13H3.4zm7 0v-5h2.2v5h-2.2z' fill='currentColor' fill-rule='evenodd'/></svg>",
+  'vertical-align-top':
+    "<svg viewBox='0 0 16 16'><path d='M15.3.1H.7a.7.7 0 1 0 0 1.4H2v13.7a.7.7 0 0 0 .7.7h3.6a.7.7 0 0 0 .7-.7V1.5h2v5.7a.7.7 0 0 0 .7.7h3.6a.7.7 0 0 0 .7-.7V1.5h1.3a.7.7 0 0 0 0-1.4zM5.6 14.5H3.4v-13h2.2v13zm7-8h-2.2v-5h2.2v5z' fill='currentColor' fill-rule='evenodd'/></svg>",
+  'warning':
+    "<svg viewBox='0 0 16 16'><path d='M8 0a8 8 0 1 1-8 8 8 8 0 0 1 8-8zm0 1.4A6.6 6.6 0 1 0 14.6 8 6.6 6.6 0 0 0 8 1.4zm.05 8.66a1 1 0 0 1 1.05 1.05 1.05 1.05 0 1 1-1.05-1.05zm.35-5.966a.7.7 0 0 1 .35.606v3.61a.7.7 0 0 1-1.4 0V4.7a.7.7 0 0 1 1.05-.606z' fill='#e43538'/></svg>",
```

Comment:
Let's not theme at the base level. I understand that this was originally done in the icon, but lets change it.
fill='#e43538' => fill='currentColor'

Reply:
ok

Reply:
Fixed